Dockerfile

```diff
@@ -4,8 +4,10 @@ EXPOSE 8080
 
 ADD sendEmail-epel-7.repo /etc/yum.repos.d/
 
+ENV NAGIOS_PLUGINS /opt/rhmap/nagios/plugins/
```

Comment:
OK, we can take it out and I'll just modify my check to hard code the path.  As long as we all agree that /opt/rhmap/nagios/plugins is where we are going to put our scripts, then its no big deal.  I made that decision on my own, so having the environment variable made it easier to change if people had other opinions.